Post/tests.py

```diff
@@ -149,3 +149,75 @@ def test_delete_post(client):
     )
 
     assert response.status_code == 302
+
+
+@pytest.mark.django_db
+def test_update_post(client):
```

Comment:
Please refer tests review in PR #132

Reply:
@Yarboa  
I added a fixture in PR #132 for user creation.
Do you want me to wait for the PR to be merge and then to use the fixture?
thanks

Reply:
Your decision here,  there is another option in case it is the same file to add it here, 
On rebase of the ongoing PR with the 1st merge it will be fixed

Reply:
Ok, I am waiting for your response about the fixture that I did in PR #132. @Yarboa